### ISSUE TITLE
Server isn't starting

### ISSUE THREAD

**Issue by dhruv3064** [insidesherpa/JPMC-tech-task-2-PY3#251]
nothing is happening while giving command in the admin cmd mode
python datafeed/server.py or 
python3 datafeed/server3.py

**Comment by Kenolise** [comment]
If you are using Python 3.x just use 

>>**python datafeed/server3.py**

without the 3 beside "python", this worked for me.

**Comment by dhruv3064** [comment]
> If you are using Python 3.x just use
> 
> > > **python datafeed/server3.py**
> 
> without the 3 beside "python", this worked for me.

**Comment by dhruv3064** [comment]
its still not responding

**Comment by rajinimasuram23** [comment]
Did u execute the server program in the " C:\Users\User_name\JPMC-tech-task-2 " directory ?

**Comment by tejas-2232** [comment]
same issue in the Linux environment. @dhruv3064 did yo solve the error?

**Comment by tejas-2232** [comment]
**update**

Running following command worked for me
> Python3 datafeed/server3.py

**Comment by tejas-2232** [comment]
Guys I'm getting error while doing 
> Npm install

Can you help me out to understand thy tyeps of errors in this install process.
Without it I can't proceed further.

**Comment by harmankaur23** [comment]
It said the same for me too, so I downloaded the node_modules folder instead

**Comment by tejas-2232** [comment]
> It said the same for me too, so I downloaded the node_modules folder instead

Node module folder is already downloaded when I cloned the repo.

**Comment by harmankaur23** [comment]
Yes the existing node_modules folder doesn't contain all the required files, there might have been an issue with npm install, so download the new one instead and replace it with the existing one

**Comment by tejas-2232** [comment]
> Yes the existing node_modules folder doesn't contain all the required files, there might have been an issue with npm install, so download the new one instead and replace it with the existing one

Ok I'll try it

**Comment by keertigupta567** [comment]
> nothing is happening while giving command in the admin cmd mode
> python datafeed/server.py or
> and after writing this "python3 datafeed/server3.py" i m getting error like
File "datafeed/server3.py", line 26, in <module>
    import dateutil.parser
ModuleNotFoundError: No module named 'dateutil'

**Comment by jc11499** [comment]
First turn off firewall. 
Try to install datautil using the troubleshoot slides given. 
If issues persist, there might be multiple python versions configured on your system. Probably python 2.x might be depreciated, so get rid of it and then try the entire process again
